test_eos-phone-home.py

```diff
@@ -0,0 +1,159 @@
+import imp
+import os
+
+import pytest
+
+HERE = os.path.dirname(__file__)
+eos_phone_home_path = os.path.join(HERE, 'eos-phone-home')
+eos_phone_home = imp.load_source("m", eos_phone_home_path)
+
+
+@pytest.fixture
+def phone_home(tmpdir):
+    return eos_phone_home.PhoneHome(True, False, root=tmpdir.strpath)
+
+
+def test_instantiate(tmpdir, phone_home):
+    assert tmpdir.strpath == phone_home._root
+
+
+def test_get_product_info_arm(tmpdir, phone_home):
+    compatible = tmpdir.join('proc', 'device-tree', 'compatible')
```

Comment:
Nit: Needs better var name as it doesn't indicate anything about what is `compatible`.

Reply:
Again that's what the pseudo-file is called, but okay.

Reply:
I have renamed this to `device_tree_product_info_file`.